app/Http/Resources/AdminTestResult.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class AdminTestResult extends JsonResource
```

Comment:
Why did you name it AdminTestResult. I can guess from the name that this is a result of a test passed by admin, but I know that this is something else...